server/index.js

```diff
@@ -103,6 +103,7 @@ async function handleAlert(location, alertType) {
         responderPushId: client.responderPushId,
         deviceName: location.displayName,
         alertType: newSession.alertType,
+        language: client.language,
```

Comment:
Do you need to sent `t` into this as well?

Reply:
Bumping Theresa's comment. Why is `t` included in `alertInfo`?